### ISSUE TITLE
SMS authentication is broken

### ISSUE THREAD

**Issue by SeanLF** [SeanLF/Tinder#1]
v2 authentication is being deprecated, and no longer works. 
v3 authentication requires gzipped content with the following mime-type: `application/x-google-protobuf`

**Comment by Jennifer-Winter** [comment]
yes sms authentication isnt working for me either. i always get sms_sent as false! :/ plssss update the code someone

**Comment by SeanLF** [comment]
In order to implement the v3 implementation of the authentication API, I'll need to set up a [protocol buffer](https://developers.google.com/protocol-buffers/docs/pythontutorial) definition, change the request's content-type to `application/x-google-protobuf`, and serialize the data using the defined protocol buffer.
